q2_longitudinal/plugin_setup.py

```diff
@@ -265,9 +266,17 @@
         **miscellaneous_parameter_descriptions,
         **shared_parameter_descriptions,
         'metric': 'Numerical metadata or artifact column to test.',
+        'baseline': (
```

Comment:
`If a "baseline" value is provided, sample differences at each state are compared against the baseline state, instead of the previous state.`

I initially interpreted this to mean that `baseline` should refer to a metadata category - I am wondering if this could be updated to make it more clear, although see my comment above about this parameter's type.

Reply:
👍 

I've added some more detail